motor/motor_gridfs.py

```diff
@@ -277,10 +275,6 @@ def __init__(self, root_collection, delegate=None, session=None,
           - `session` (optional): a
             :class:`~pymongo.client_session.ClientSession` to use for all
             commands
-          - `disable_md5` (optional): When True, an MD5 checksum will not be
```

Comment:
Here and elsewhere please add a `.. versionchanged:: 3.0` to document that disable_md5 was removed.

Reply:
Will do

Reply:
Done